.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to WordPress.org
+on:
+  release:
+    types: [published]
+
+  workflow_dispatch:
+
+jobs:
+  tag:
+    name: New release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Install dependencies
+      uses: php-actions/composer@v5
+      with:
+        php_version: 7
+        version: 1
+        command: install
+        args: --no-dev
+    - name: Build
+      run: |
+        npm install
+        npm run build
```

Comment:
I'm not 100% sure how this is going to behave, since we currently check the Algolia libraries (PHP and JS) into version control. 🤔

Reply:
I agree with this assessment. Thank you @richaber!